src/dmd/dtoh.d

```diff
@@ -193,14 +188,14 @@ private void initialize()
     }
 }
 
-void genCppHdrFiles(ref Modules ms)
+extern(C++) void genCppHdrFiles(ref Modules ms)
 {
     initialize();
 
     OutBuffer buf;
-    buf.writestring("#pragma once\n");
+    buf.printf("// Automatically generated by %s Compiler v%d\n", __VENDOR__.ptr, __VERSION__);
```

Comment:
Wont this use the info of the compiler used to compile the compiler?

Reply:
I forget that dmd doesn't bootstrap itself by default.  It's likely far too late for me to think anymore, and now you mention it, using internal global params directly makes more sense.